ts/IterableInboxMessageList.tsx

```diff
@@ -52,18 +55,56 @@ const IterableInboxMessageList = ({
       )
    }
 
+   function getInAppMessageFromViewToken(viewToken: ViewToken): IterableInAppMessage {
+      return IterableInAppMessage.fromInApp(viewToken.item["inAppMessage"] as IterableInAppMessage)
+   }
+
+   // function getInAppMessagesFromViewTokens(viewTokens: Array<ViewToken>): Array<IterableInAppMessage> {
+   //    return viewTokens.map(getInAppMessageFromViewToken)
+   // }
+
+   function getRowInfosFromViewTokens(viewTokens: Array<ViewToken>): Array<InboxImpressionRowInfo> {
+      return viewTokens.map(
+         function(viewToken) {
```

Comment:
Can this function be moved outside? If inline, I think keyword 'function' wont be needed

Reply:
May be its just cosmetic change

Reply:
I'd prefer not to move it outside for now because it's pretty parasitic (only works within this component).

I also can't remove `function`, it causes errors.